default.config.yml

```diff
@@ -215,6 +216,7 @@ extra_packages:
 extra_security_enabled: false
 
 drush_version: "master"
+drush_path: "{{ drupal_composer_install_dir }}/vendor/drush/drush/drush"
```

Comment:
My task idea didn't work. If I set the variable it's not possible to override it in `config.yml`. Any ideas?